packages/now-node/src/index.ts

```diff
@@ -74,6 +74,8 @@ function isPortInfo(v: any): v is PortInfo {
   return v && typeof v.port === 'number';
 }
 
+const baseDir = '/';
```

Comment:
We could pass in the repository root directory and use that as the base, but technically Node.js will resolve all the way up to the system root as described here:

https://nodejs.org/docs/latest-v12.x/api/modules.html#modules_loading_from_node_modules_folders